server/utils/vulnerabilitySummary.ts

```diff
@@ -0,0 +1,35 @@
+import type { Environment } from '../data/strapiApiTypes'
+import type { Summary } from '../data/converters/modelTypes'
+import type { VeracodeResultsSummary } from '../data/modelTypes'
+
+export const getProductionEnvironment = (envs?: Environment[]): Environment | undefined => {
+  return envs?.find(environment => environment.name === 'prod')
+}
+
+export const countTrivyHighAndCritical = (summary?: Summary): number => {
+  if (!summary) return 0
+  const categories: Array<'os-pkgs' | 'lang-pkgs'> = ['os-pkgs', 'lang-pkgs']
+  let total = 0
+  categories.forEach(category => {
```

Comment:
To make this a bit more functional programming, you could use `.reduce` then it'd be `const total = categories.reduce(...., 0)` maybe? Sorry I have a dislike of `let` 😆